src/main/c/powerwaf_jni.c

```diff
@@ -603,8 +592,7 @@ JNIEXPORT jlong JNICALL Java_io_sqreen_powerwaf_Additive_initAdditive(
         return 0L;
     }
 
-    ddwaf_context context = ddwaf_context_init(
-            nat_handle, is_byte_buffers ? NULL : ddwaf_object_free);
+    ddwaf_context context = ddwaf_context_init(nat_handle);
```

Comment:
You removed `is_byte_buffers` so now, tests are failing with the error:
```
unused parameter 'is_byte_buffers'
```
in function definition (line [powerwaf_jni.c:586](https://github.com/sqreen/AgentJavaNative/blob/06a51146d1da3da14ecbc8357b6ce1883bd75d48/src/main/c/powerwaf_jni.c#L586))